module/product/src/Application/Controller/Api/Attribute/UpdateProductsAttributesAction.php

```diff
@@ -72,14 +72,13 @@ public function __construct(
      * @SWG\Parameter(
      *     name="body",
      *     in="body",
-     *     description="Add category ID",
      *     required=true,
-     *     @SWG\Schema(ref="#/definitions/product_category")
+     *     @SWG\Schema(ref="#/definitions/attribute")
      * )
      *
      * @SWG\Response(
      *     response=204,
-     *     description="Update mass pproducts attribtes",
+     *     description="Update mas products attribtes",
```

Comment:
```suggestion
     *     description="Update mass products attribtes",
```
BTW didn't we use `batch` convention?